examples/src/main/java/io/opencensus/examples/zpages/ZPagesTester.java

```diff
@@ -16,14 +16,87 @@
+  private static final StatsRecorder statsRecorder = Stats.getStatsRecorder();
+
+  private static final String SPAN_NAME = "SampleSpan";
+
   /** Main method. */
   public static void main(String[] args) throws Exception {
+    Tracing.getExportComponent()
+        .getSampledSpanStore()
+        .registerSpanNamesForCollection(Collections.singletonList(SPAN_NAME));
+    RpcViews.registerAllViews(); // Use old RPC constants to get interval stats.
+    SpanBuilder spanBuilder =
+        tracer.spanBuilder(SPAN_NAME).setRecordEvents(true).setSampler(Samplers.alwaysSample());
+
+    try (Scope scope = spanBuilder.startScopedSpan()) {
+      tracer.getCurrentSpan().addAnnotation("Starts recording.");
+      MeasureMap measureMap =
+          statsRecorder
+              .newMeasureMap()
+              // Client measurements.
+              .put(RpcMeasureConstants.RPC_CLIENT_STARTED_COUNT, 1)
+              .put(RpcMeasureConstants.RPC_CLIENT_FINISHED_COUNT, 1)
+              .put(RpcMeasureConstants.RPC_CLIENT_ROUNDTRIP_LATENCY, 1.0)
+              .put(RpcMeasureConstants.RPC_CLIENT_REQUEST_COUNT, 1)
+              .put(RpcMeasureConstants.RPC_CLIENT_RESPONSE_COUNT, 1)
+              .put(RpcMeasureConstants.RPC_CLIENT_REQUEST_BYTES, 1e5)
+              .put(RpcMeasureConstants.RPC_CLIENT_RESPONSE_BYTES, 1e5)
+              .put(RpcMeasureConstants.RPC_CLIENT_UNCOMPRESSED_REQUEST_BYTES, 1e5)
+              .put(RpcMeasureConstants.RPC_CLIENT_UNCOMPRESSED_RESPONSE_BYTES, 1e5)
+              // Server measurements.
+              .put(RpcMeasureConstants.RPC_SERVER_STARTED_COUNT, 1)
+              .put(RpcMeasureConstants.RPC_SERVER_FINISHED_COUNT, 1)
+              .put(RpcMeasureConstants.RPC_SERVER_SERVER_LATENCY, 1.0)
+              .put(RpcMeasureConstants.RPC_SERVER_REQUEST_COUNT, 1)
+              .put(RpcMeasureConstants.RPC_SERVER_RESPONSE_COUNT, 1)
+              .put(RpcMeasureConstants.RPC_SERVER_REQUEST_BYTES, 1e5)
+              .put(RpcMeasureConstants.RPC_SERVER_RESPONSE_BYTES, 1e5)
+              .put(RpcMeasureConstants.RPC_SERVER_UNCOMPRESSED_REQUEST_BYTES, 1e5)
+              .put(RpcMeasureConstants.RPC_SERVER_UNCOMPRESSED_RESPONSE_BYTES, 1e5);
+      measureMap.record(
+          tagger
+              .currentBuilder()
+              .put(RpcMeasureConstants.RPC_STATUS, TagValue.create("OK"))
+              .build());
+      MeasureMap measureMapErrors =
+          statsRecorder
+              .newMeasureMap()
+              .put(RpcMeasureConstants.RPC_CLIENT_ERROR_COUNT, 1)
+              .put(RpcMeasureConstants.RPC_SERVER_ERROR_COUNT, 1);
+      measureMapErrors.record(
+          tagger
+              .currentBuilder()
+              .put(RpcMeasureConstants.RPC_STATUS, TagValue.create("UNKNOWN"))
+              .build());
+
+      Thread.sleep(200); // sleep for fake work.
+      tracer.getCurrentSpan().addAnnotation("Finish recording.");
+    }
+
     ZPageHandlers.startHttpServerAndRegisterAll(8080);
```

Comment:
Shouldn't the zpages be started at the beginning of the method, since they are usually registered at program initialization?

Reply:
That's right, done.